tensorboard/plugins/hparams/summary_v2.py

```diff
@@ -55,12 +57,13 @@ def hparams(hparams, start_time_secs=None):
   """
   pb = hparams_pb(
       hparams=hparams,
+      trial_id=trial_id,
       start_time_secs=start_time_secs,
   )
   return _write_summary("hparams", pb)
 
 
-def hparams_pb(hparams, start_time_secs=None):
+def hparams_pb(hparams, trial_id=None, start_time_secs=None):
```

Comment:
Probably no action required: if we shipped this API as part of 1.14, and if a user don't use the kwarg, this change will break them, right?

Reply:
Correct; you’re supposed to use use the kwarg.

(But we wouldn’t ship this as part of 1.14 anyway, because it’s a new
feature.)

Reply:
Maybe we can change these to be keyword-only arguments once we drop
Python 2 support, and likewise with `TBContext` and friends.